lib/cylc/scheduler.py

```diff
@@ -1574,6 +1392,12 @@ def run(self):
 
             self.pool.process_queued_task_messages()
             self.pool.process_queued_task_event_handlers()
+            self.process_command_queue()
```

Comment:
I think it is because some tests are failing because `process_tasks`, etc can take a long time. For some reason, adding this repetition makes the tests more stable.